CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+## When to submit an ADR
+
+The purpose of an ADR is to record an over-arching technical decision made by the development team in the Archivematica 
+project. An ADR is akin to a policy in other contexts- it describes a way that future features or changes should be 
+implemented, rather than describing the features themselves (to submit a feature idea, head to the Issues repo).  
+Developers will use ADRs to understand how code should be written and analysts and end-users to understand implementation 
```

Comment:
I wonder here about tense and specificity. Perhaps `Developers, analysts, and end-users, will use ADRs to understand how code was written and implementation decisions that may be visible to the user.`?

Reply:
This is much more clear- made this change also.